src/test/java/org/folio/circulation/domain/policy/PeriodTest.java

```diff
@@ -0,0 +1,31 @@
+package org.folio.circulation.domain.policy;
+
+import junitparams.JUnitParamsRunner;
+import junitparams.Parameters;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static org.junit.Assert.*;
+
+@RunWith(JUnitParamsRunner.class)
+public class PeriodTest {
+
+  @Test
+  @Parameters
+  public void toMinutes(String interval, Integer duration, int expectedResult) {
+    assertEquals(expectedResult, Period.from(duration, interval).toMinutes());
+  }
+
+  private Object[] parametersForToMinutes() {
+    return new Object[]{
+      new Object[] {"Minutes", 6, 6},
+      new Object[] {"Hours", 5, 300},
+      new Object[] {"Days", 4, 5760},
+      new Object[] {"Weeks", 3, 30240},
+      new Object[] {"Months", 2, 89280},
+      new Object[] {"Random", 10, 10},
+      new Object[] {null, 10, 0},
+      new Object[] {"Minutes", null, 0},
```

Comment:
Personally, I think these two represent a different kind of scenario, where the period is invalid rather than the calculation of the minutes within a period.

Reply:
done